lib/network/modules/components/Node.js

```diff
@@ -527,17 +527,15 @@ class Node {
     return this.options.value;
   }
 
-
   /**
    * Get the current dimensions of the label
    *
    * @return {rect}
    */
-  getLabelSize() {
-    return this.labelModule.size();
+  getLabelDimensions() {
+    return this.labelModule.getDimensions();
   }
```

Comment:
Unless you need it delete it. The only thing it was doing before was throwing errors. Nobody complained so nobody needs it.